web_transport/sdk/api/owt/quic/web_transport_session_interface.h

```diff
@@ -30,6 +30,10 @@ class OWT_EXPORT WebTransportSessionInterface {
   virtual MessageStatus SendOrQueueDatagram(uint8_t* data, size_t length) = 0;
   // Get connection stats.
   virtual const ConnectionStats& GetStats() = 0;
+  // Close a WebTransport session. `code` is the error code communicated with
+  // peer, `reason` is a pointer to a UTF-8 encoded null terminated string, its
+  // length should not exceed 1024.
+  virtual void Close(uint32_t code, char* reason) = 0;
```

Comment:
How do you prevent potential buffer overflow attack here?=

Reply:
The lower level implementation creates buffer based on `reason`'s size, not hard coded 1024. But I'll follow the spec to truncate `reason` if it's exceed 1024.